README.md

```diff
@@ -90,6 +90,12 @@ By default, only messages with `type: "warning"` warnings are logged. (These are
 
 For example, `function(message) { return true }` will only every message, regardless of the plugin or whether it's a warning or not.
 
+**clearAllMessages** (function)
```

Comment:
@kandebr I'm not insisting either, I'm just wondering if there *is* a use-case for the filter function. If there *is* a use-case we can do it; if we can't think of one, let's just add the boolean.